app/controllers/v0/institutions_controller.rb

```diff
@@ -84,8 +84,28 @@ def facets
         principles_of_excellence: search_results.filter_count(:poe),
         eight_keys_to_veteran_success: search_results.filter_count(:eight_keys)
       }
+      add_active_search_facets(result)
     end
 
+    # rubocop:disable Metrics/CyclomaticComplexity, Style/GuardClause
+    def add_active_search_facets(raw_facets)
+      if @query[:state].present?
+        key = @query[:state].downcase
+        raw_facets[:state][key] = 0 unless raw_facets[:state].key? key
+      end
+      if @query[:type].present?
+        key = @query[:type].downcase
+        raw_facets[:type][key] = 0 unless raw_facets[:type].key? key
+      end
+      if @query[:country].present?
+        key = @query[:country].upcase
+        raw_facets[:country] << { name: key, count: 0 } unless
+          raw_facets[:country].any? { |c| c[:name] == key }
+      end
+      raw_facets
+    end
+    # rubocop:enable Metrics/CyclomaticComplexity, Style/GuardClause
```

Comment:
Yeah, although I think there's an argument to be made that a query with a search parameter should include that parameter in the facets - not just because it happens to make our frontend work but because it's making the output metadata reflect the input parameters of the request.